packages/config/src/layer2s/common/exits.ts

```diff
@@ -75,15 +75,15 @@ const STARKEX_FORCED: ProjectTechnologyChoice = {
   references: [
     {
       text: 'Forced Operations - StarkEx documentation',
-      href: 'https://docs.starkware.co/starkex-v3/starkex-deep-dive/regular-flows/flows-for-off-chain-accounts/forced-operations',
+      href: 'https://docs.starkware.co/starkex/spot/shared/README-forced-operations.html',
     },
     {
       text: 'Forced Withdrawal - StarkEx documentation',
-      href: 'https://docs.starkware.co/starkex-v3/starkex-deep-dive/smart-contracts-1/in-spot-trading/in-perpetual-trading',
+      href: 'https://docs.starkware.co/starkex/perpetual/perpetual-trading-forced-withdrawal-and-forced-trade.html',
```

Comment:
```suggestion
      href: 'https://docs.starkware.co/starkex/perpetual/perpetual-trading-forced-withdrawal-and-forced-trade.html#forced_withdrawal',
```

Reply:
Sorry English

Ok, my confusion is also on the same links for Forced Perpetual and Trade, but with different name?
I don't know which one will be better to use, I made some changes waiting for your reply...in line 78

https://docs.starkware.co/starkex/perpetual/shared/README-forced-operations.html#forced_operation_flows_perpetual

https://docs.starkware.co/starkex/spot/shared/README-forced-operations.html#forced_operation_flows_spot